scripts/install

```diff
@@ -148,8 +142,12 @@ ynh_use_nodejs
 # Setup Kresus postgresql database
 #=================================================
 
+db_name="$app"
+db_user="$app"
+
 ynh_psql_test_if_first_run
-ynh_psql_setup_db "$db_user" "$db_name" "$db_pwd"
+ynh_psql_setup_db "$db_user" "$db_name"
+db_pwd=$(ynh_app_setting_get "$app" psqlpwd)
```

Comment:
How does that work? The setting was not set already? Or does `ynh_psql_setup_db` set it globally?

Reply:
yes, ynh_psql_setup_db does it for us: https://github.com/YunoHost/yunohost/blob/stretch-unstable/data/helpers.d/postgresql#L210